packages/react/ds/src/form-group/form-group.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export type FormGroupProps = React.HTMLAttributes<HTMLDivElement> & {
+  hasError?: boolean;
+  children: React.ReactNode;
+};
+
+export const FormGroup = React.forwardRef<HTMLDivElement, FormGroupProps>(
+  ({ hasError = false, children, ...props }, ref) => {
```

Comment:
do we need a default value here ? the `hasError` is optional and so it will be falsy when we don't declare it

Reply:
Good catch!